tests/encapsed/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -117,9 +117,11 @@ $code = <<<EOF
 \\$kernel = unserialize('$kernel');
 \\$request = unserialize('$request');
 EOF;
-$content = sprintf(<<<EOF
+$content = sprintf(
+    <<<EOF
 foo
 EOF
+
```

Comment:
This newline is unexpected though, right?

Reply:
correct - not sure why its happening though... I can try and investigate

Reply:
There is some special logic that adds linebreaks after heredoc/nowdoc nodes, because before we had issues with code like `EOF;` or `EOF]` which would break.